checkov/terraform/checks/resource/aws/SecretManagerSecret90days.py

```diff
@@ -0,0 +1,25 @@
+
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+
+
+class SecretManagerSecret90days(BaseResourceCheck):
+
+    def __init__(self):
+        name = "Ensure Secrets Manager secrets should be rotated within a specified number of days"
```

Comment:
```suggestion
        name = "Ensure Secrets Manager secrets should be rotated within 90 days"
```
wouldn't it be better to just add the number here, so it is more clear what we expect?